src/observablePlaylist.ts

```diff
@@ -125,3 +147,61 @@ export namespace ObservablePlaylist {
     nextTrackInPlaylist: Subject<Track | undefined>
   }
 }
+
+function printQueue(start: number, queue: ObservablePlaylist.Track[], currentQueueIndex: number, currentPage: number) {
+  const printedQueue: string[] = []
+  const end = start + pageSize
+  let j = 0
+  for (let i = start; i <= end; i++) {
+    const curr = queue[i]
+    if (curr) {
+      if (curr.removed) {
+        printedQueue[j++] = `\`${i})\` ~~${curr.name}~~`
+      } else if (i == currentQueueIndex) {
+        printedQueue[j++] = `\`${i})\` ${curr.name} ← Currently playing`
+      } else {
+        printedQueue[j++] = `\`${i})\` ${curr.name}`
+      }
+    }
+  }
+
+  let embed = new MessageEmbed()
+    .setColor('#ffb81f')
+    .setTitle('Queue')
+    .setDescription(printedQueue.join('\n'))
+    .setTimestamp()
+  let reactions: string[] = []
+
+  if (currentPage > 1) {
+    reactions.push('⏪')
+  }
+
+  if (currentPage > 0) {
+    reactions.push('◀')
+  }
+
+  if (end < queue.length) {
+    reactions.push('▶')
+  }
+
+  if (pageSize * 2 < queue.length - start) {
+    reactions.push('⏩')
+  }
+
+  if (reactions.length > 0) {
+
+    embed.addField('\n' + nrOfTracksLeftString(queue, end) + ' more track(s) in queue.', 'During the current session ' + queue.length + ' tracks have been added to the queue in total')
```

Comment:
Föredrar string-interpolation
```suggestion
    embed.addField(`\n${nrOfTracksLeftString(queue, end)} more track(s) in queue.`, `During the current session ${queue.length} tracks have been added to the queue in total`)

```

Reply:
Same nu när jag ser det 😄